components/camel-aws/camel-aws-secrets-manager/src/main/docs/aws-secrets-manager-component.adoc

```diff
@@ -270,6 +270,66 @@ where `camel.vault.aws.refreshEnabled` will enable the automatic context reload,
 
 Note that `camel.vault.aws.secrets` is not mandatory: if not specified the task responsible for checking updates events will take into accounts or the properties with an `aws:` prefix.
 
+=== Automatic Camel context reloading on Secret Refresh with Eventbridge and AWS SQS Services
+
+There is also the possibility of using AWS Eventbridge in combination with AWS SQS Service.
+
+The needed bits on the AWS side are the following:
```

Comment:
```
On the AWS side, the following resources need to be created:

- an AWS Couldtrail trail
- an AWS SQS Queue
- an Eventbridge rule of this kind
```

Reply:
Btw, I'm not sure what `this kind` refers too, maybe it needs to be a bit more explicit ?

Reply:
Ah, what about `of the following kind` ?

Reply:
Done